src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -178,23 +178,23 @@ class PeerActor(
 
       daoBlockHeaderOpt match {
         case Some(_) if daoForkValidator.validate(msg).isEmpty =>
-          log.info("Peer is running the ETC chain")
+          log.warning("Peer is running the ETC chain")
           context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber, None).receive
 
         case Some(_) if nodeStatusHolder().blockchainStatus.totalDifficulty < daoForkBlockTotalDifficulty =>
-          log.info("Peer is not running the ETC fork, but we're not there yet. Keeping the connection until then.")
+          log.warning("Peer is not running the ETC fork, but we're not there yet. Keeping the connection until then.")
```

Comment:
This also sounds like something that could happen in the normal flow of events ... info?